### PR TITLE
PAD-1531: list spaces command integration

### PR DESCRIPTION
**Pull request by ashishkubaghel**
#### Description

Added a new command for list to get all spaces from the provided Team.

#### Checklist

- [x] I have self-reviewed this PR
- [x] I have tested the change and proved that it works in different scenarios
- [ ] I have updated docs if needed

### CODE REVIEW

File: src/content/manager/space.manager.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    public set name(value: string) {
+        this._name = value;
+    }
+
+    public getConfig(): ManagerConfig {
+        return {
+            findAllUrl: this.profile.team.replace(/\/?$/, SpaceManager.BASE_URL),
+            onFindAll: (data: SaveSpace[]) => this.listSpaces(data),
```
Comment (pmoranjusdado) [inline src/content/manager/space.manager.ts:32]: Would it be possible to use here already existing `SaveContentNode` interface?

**Review by pmoranjusdado — COMMENTED** [review]
Thanks for the PR Ashish! Overall LGTM, Added one point to check!

File: src/content/factory/space-manager.factory.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {SpaceManager} from "../manager/space.manager";
+
+export class SpaceManagerFactory {
+
+    public createManager(
+        id?: string,
```
Comment (promeris) [inline src/content/factory/space-manager.factory.ts:6]: I don't see the point of these parameters as they aren't provided or used anywhere. Can we just remove them?

**Review by promeris — COMMENTED** [review]
Please also bump the version in package.json before merging.

**Comment by ashishkubaghel** [comment]
Thank you @promeris , @pmoranjusdado  for your suggestions. Could you please re-review?

**Comment by ashishkubaghel** [comment]
@promeris Could you please re-review and let us know if you have any more suggestions? or looks good to proceed.

**Review by promeris — APPROVED** [review]

**Review by frkntrcnn — APPROVED** [review]
LGTM 👍

**Comment by osyniakov** [comment]
@ashishkubaghel could you please create a follow-up PR to update [documentation](https://github.com/celonis/content-cli/blob/master/DOCUMENTATION.md#list-all-packages-in-studio)?

**Pull request merged**